ui/src/index.tsx

```diff
@@ -8,7 +8,7 @@ ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root'),
```

Comment:
is this `,` required by eslint?

Reply:
This was the prettier issue, I have changed the config of prettier :+1: